index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en-US">
```

Comment:
![image](https://user-images.githubusercontent.com/35924318/71359606-afeb2700-2595-11ea-920c-424eb7b8a824.png)
keep in mind responsiveness